sections/stats/OpenInterest.tsx

```diff
@@ -0,0 +1,302 @@
+import { useEffect, useRef, useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useRecoilValue } from 'recoil';
+
+import AAVEIcon from 'assets/png/currencies/sAAVE.png';
+import APEIcon from 'assets/png/currencies/sAPECOIN.png';
+import AVAXIcon from 'assets/png/currencies/sAVAX.png';
+import BTCIcon from 'assets/png/currencies/sBTC.png';
+import DYDXIcon from 'assets/png/currencies/sDYDX.png';
+import ETHIcon from 'assets/png/currencies/sETH.png';
+import EURIcon from 'assets/png/currencies/sEUR.png';
+import LINKIcon from 'assets/png/currencies/sLINK.png';
+import MATICIcon from 'assets/png/currencies/sMATIC.png';
+import SOLIcon from 'assets/png/currencies/sSOL.png';
+import UNIIcon from 'assets/png/currencies/sUNI.png';
+import XAGIcon from 'assets/png/currencies/sXAG.png';
+import XAUIcon from 'assets/png/currencies/sXAU.png';
```

Comment:
Should be able to get these from the config without importing every one individually.